features/other_b/skin.feature

```diff
@@ -41,8 +45,10 @@ Feature: creating and editing skins
    When I am logged in as "skinner"
     And I follow "public skin"
    Then I should see "The skin public skin has been set. This will last for your current session."
-
-
+   When I am logged in as an admin
+     And I follow "Approved Skins"
+     And I check "make_unin_chooser_public_skin"
```

Comment:
Cucumbers are testing user-facing things, so can you use the user-facing test here? It's "Not In Chooser." (This wouldn't work if we had multiple skins on the page, but we only create one for this scenario, so it should work fine.)